src/main/java/wordcloud/nlp/normalize/BubbleTextNormalizer.java

```diff
@@ -0,0 +1,25 @@
+package wordcloud.nlp.normalize;
+
+/**
+ * Replaces the characters a-zA-Z with their bubble pendants ⓐ-ⓩⒶ-Ⓩ
+ * 
+ * @author &#64;wolfposd
+ *
+ */
+public class BubbleTextNormalizer implements Normalizer {
+
+    private static String normal = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    private static String bubbles = "ⓐⓑⓒⓓⓔⓕⓖⓗⓘⓙⓚⓛⓜⓝⓞⓟⓠⓡⓢⓣⓤⓥⓦⓧⓨⓩⒶⒷⒸⒹⒺⒻⒼⒽⒾⒿⓀⓁⓂⓃⓄⓅⓆⓇⓈⓉⓊⓋⓌⓍⓎⓏ";
```

Comment:
I would like you to render an image using this text normalizer and stick it with the other outputs